docs/3.x.x/getting-started/install-requirements.md

```diff
@@ -0,0 +1,197 @@
+# Installation Requirements
+
+### 👋 Welcome onboard!
+
+Welcome to the open source [headless CMS](https://strapi.io) developers love. Our users love Strapi because it is open source, MIT licensed, fully customizable and based on nodejs. Strapi lets you manage your content and distribute it anywhere.
+
+This page covers installing the [basic installation requirements for Strapi](#basic-installation-requirements).
+
+(After you install these requirements, the [Quick Start](/3.x.x/getting-started/quick-start.html) guide will walk you through how to: [create a project](quick-start.html#_1-create-a-project), [create an admin user](quick-start.html#_2-create-an-admin-user), [create a content type](quick-start.html#_3-create-a-content-type), [manage and add data to content type](quick-start.html#_4-manage-and-add-data-to-content-type), [set roles and permissions](quick-start.html#_5-set-roles-and-permissions) and [consume the content type API](quick-start.html#_6-consume-the-content-type-api).)
```

Comment:
Yes, I noticed that before. It's because the previous section is not large enough. Not enough content. It is like a vuepress bug.  
If we add some images, this will correct itself.

Reply:
This is now okay. :)